libkbfs/folder_branch_ops.go

```diff
@@ -6020,6 +6180,12 @@ func (fbo *folderBranchOps) doFastForwardLocked(ctx context.Context,
 		return err
 	}
 
+	latestRootBlockFetch := fbo.kickOffRootBlockFetch(ctx, currHead)
+	err = fbo.waitForRootBlockFetch(ctx, currHead, latestRootBlockFetch)
```

Comment:
For synced folders, this will block reading the root and everything under it until it's synced with remote (via `registerAndWaitForUpdates`. Is that desired?

Reply:
> everything under it

?  I don't think that's the case, it should only wait for the single root block to be fetched.  If I'm wrong, please let me know.

Reply:
Ah nvmind, I misread this as `WaitChannelForBlockPrefetch`.